### PR TITLE
Update with new consumed Weight structures

### PR DESCRIPTION
**Pull request by jacogr**
Pulls in https://github.com/paritytech/substrate/pull/6629 - thanks to @bkchr for the ping

### CODE REVIEW

File: packages/metadata/src/v12/static-substrate.json [hunk 1]
```diff
@@ -462,14 +462,6 @@
                 " The maximum number of blocks to allow in mortal eras."
               ]
             },
-            {
```
Comment (jacogr) [inline packages/metadata/src/v12/static-substrate.json:465]: This breaks a couple of things, both in the API and the apps UI.

File: packages/metadata/src/v12/static-substrate.json [hunk 2]
```diff
@@ -462,14 +462,6 @@
...
-              "name": "MaximumBlockWeight",
-              "type": "Weight",
-              "value": "0x00204aa9d1010000",
-              "documentation": [
-                " The maximum weight of a block."
-              ]
-            },
```
Comment (jacogr) [inline packages/metadata/src/v12/static-substrate.json:472]: This breaks a couple of things, both in the API and the apps UI.

**Comment by pepyakin** [comment]
Is there an estimate when this can get onto npm? Does it happen automatically or manually?

**Comment by jacogr** [comment]
beta versions of the API is published from CI, see https://github.com/polkadot-js/api/actions

**Pull request merged**